### PR TITLE
Added Simon game to the collection

### PR DESCRIPTION
**Pull request by Aman-kumar001**
## Feature added.
  - #619

Closes: #619

### Describe the changes you've made
Added Simon's game to the collection. A game that tests the memory power of an individual.

### Describe if there is any unusual behavior of your code(Write `NA` if there isn't)
NA

### Checklist:
Example how to mark a checkbox:-
- [x] Created a sequence to store the answer.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
Simons_s_Game

### CODE REVIEW

**Review by Vishal-raj-1 — APPROVED** [review]
LGTM,Just change bg.png name to yourprojectname-bg.png . And squash your commit into one single commit, Then It will be ready for merged

**Comment by Aman-kumar001** [comment]
Resolved Conflicts which was occurred due to clashing of the name of 2 files.

**Comment by Aman-kumar001** [comment]
done with squashing of commits. Please add a label to this PR.

File: public/simon.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Amatic+SC:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
```
Comment (991rajat) [inline public/simon.html:9]: Properly link the css

File: public/simon.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <div class="verti">
+            <div class="b2 box shadow" id="c2"></div>
+            <div class="b3 box shadow" id="c3"></div>
+        </div>
+        <div class="b4 box shadow" id="c4"></div>
+    </div>
+</body>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+<script src="script.js"></script>
```
Comment (991rajat) [inline public/simon.html:26]: properly link js

**Review by 991rajat — CHANGES_REQUESTED** [review]
Also it would be great if you could reduce the size of png.

File: public/simon.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <div class="verti">
+            <div class="b2 box shadow" id="c2"></div>
+            <div class="b3 box shadow" id="c3"></div>
+        </div>
+        <div class="b4 box shadow" id="c4"></div>
+    </div>
+</body>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+<script src="simon.js"></script>
```
Comment (991rajat) [inline public/simon.html:26]: Fix the path 
it's ../assets/js/simon.js

File: public/simon.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Amatic+SC:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="simon.css">
```
Comment (991rajat) [inline public/simon.html:9]: Fix the path
it's ../assets/css/simon.css

**Review by 991rajat — CHANGES_REQUESTED** [review]
Css and JS not linked properly. On your side is it okay?

File: public/simon.html (hunk 4 above)
Comment (Aman-kumar001) [reply public/simon.html:9]: I was not aware of this. Thankyou.

**Comment by Aman-kumar001** [comment]
> 
> 
> Css and JS not linked properly. I find out path is the issue. On your side is it okay?

It should work fine now.

**Review by 991rajat — APPROVED** [review]
Good Work!

**Comment by 991rajat** [comment]
Congratulations on merging of first PR. :tada: 
Keep Contributing.

**Comment by Aman-kumar001** [comment]
> 
> 
> Congratulations on merging of first PR. 🎉
> Keep Contributing.



> 
> 
> Congratulations on merging of first PR. 🎉
> Keep Contributing.

Thank you.
i have a request. can you please ad label to this.

**Comment by 991rajat** [comment]
Thanks for reminding.
